indico/web/templates/_sortable_list.html

```diff
@@ -41,28 +41,29 @@
     </div>
 {%- endmacro %}
 
-{% macro sortable_lists(enabled_title, enabled_items, disabled_title, disabled_items, classes="", draggable=true,
+{% macro sortable_lists_any(lists, classes="", draggable=true, toggle_enabled=false,
                         id=none, invisible_handle=false) -%}
     {# Caller is called with an item and must render the action buttons. #}
     {% set _caller = caller|default(none) %}
     <div class="sortable-list js-sortable-list-widget {{ classes }}"
          {% if id %}id="{{ id }}"{% endif %}
          {% if not draggable %}data-disable-dragging{% endif %}>
+        {% for list in lists %}
         <div>
-            <div class="titled-rule">{{ enabled_title }}</div>
-            <ul class="enabled">
-                {% call(item) sortable_items(enabled_items, draggable, true, invisible_handle) %}
-                    {{ _caller(item) if _caller else '' }}
-                {% endcall %}
-            </ul>
-        </div>
-        <div>
-            <div class="titled-rule">{{ disabled_title }}</div>
-            <ul class="disabled">
-                {% call(item) sortable_items(disabled_items, draggable, true, invisible_handle) %}
+            <div class="titled-rule">{{ list.0 }}</div>
```

Comment:
I'd rather use bracket notation for numeric keys. It looks really strange with dot notation ;)